lib/models/task.dart

```diff
@@ -0,0 +1,25 @@
+class Task {
+  String? id;
+  String name;
+
+  Task({id, required this.name});
+
+  factory Task.fromJson(Map<String, dynamic> json) {
+    return Task(
+      id: json['id'].toString(),
+      name: json['name'].toString(),
+    );
+  }
+
+  String getName() {
+    return name;
+  }
+
+  String? getId() {
+    return id;
+  }
+
+  void setName(String name) {
+    this.name = name;
+  }
```

Comment:
Typically I don't write getters and setters for Dart models. The rules of encapsulation work differently in Dart than they do in Java so the value in using a getter/setter is different. If this works for your brain, this is fine. I personally prefer

```dart
Center(
  child: Text(task.name),
),
```

over

```dart
Center(
  child: Text(task.getName()),
),
```

Reply:
Done!